### ISSUE TITLE
Wallet will not sync and stuck at 65 block for last 3 days

### ISSUE THREAD

**Issue by Thatonegod** [ethereum/mist#3860]
Hi everyone I have tried everything I can think of and read every scrap of information I could get my hands on my wallet will not sync since it gets stuck at 65 blocks. ive tried to use the beta light version that takes around 2 hours to load and it dose give me the balance but I am unable to send any ethereum I have bin dealing with this for around 3 days now Please help!!!

* [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.0.0`
OS & Version: windows/linux/osx
Node version: `geth 0.0.0`
Number of blocks synchronized: 0
```

**Comment by lorezyra** [comment]
https://github.com/ethereum/mist/issues/2466

**Comment by Thatonegod** [comment]
I am running Windows X and I am using the latest wallet that I downloaded last night

**Comment by englsalsa** [comment]
same problem win7 x64 Ethereum-Wallet-installer-0-10-0

**Comment by limitingKano** [comment]
I'm in the same position, I've tried sending my funds out to a more stable wallet but the transaction just sits in latest transaction and never gets confirmed. Etherscan has no record of the transaction?

**Comment by EthereumMagic** [comment]
Do you sync with light client alone or have to do something else?

**Comment by ryanio** [comment]
Hey, we just released a new version (`0.11.0`) that has new syncing features - please give it a try! If you have any new problems that you can reproduce, please create a new issue with the details on how 
we can help. Thanks!
